src/RecoveryServices/RecoveryServices.Backup/Cmdlets/ProtectionPolicy/SetAzureRmRecoveryServicesBackupProtectionPolicy.cs

```diff
@@ -32,28 +32,42 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
     [Cmdlet("Set", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RecoveryServicesBackupProtectionPolicy",SupportsShouldProcess = true), OutputType(typeof(JobBase))]
     public class SetAzureRmRecoveryServicesBackupProtectionPolicy : RSBackupVaultCmdletBase
     {
+        public const string ModifyPolicyParamSet = "ModifyPolicyParamSet";
+        public const string FixInconsistentPolicyParamSet = "FixPolicyParamSet";
         /// <summary>
         /// Policy object to be modified
         /// </summary>
         [Parameter(Position = 1, Mandatory = true, HelpMessage = ParamHelpMsgs.Policy.ProtectionPolicy,
-            ValueFromPipeline = true)]
+            ValueFromPipeline = true, ParameterSetName = ModifyPolicyParamSet)]
+        [Parameter(Position = 1, Mandatory = true, HelpMessage = ParamHelpMsgs.Policy.ProtectionPolicy,
+            ValueFromPipeline = true, ParameterSetName = FixInconsistentPolicyParamSet)]
```

Comment:
If the parameter appears in all the parameter sets, you don't have to specify it twice. In fact, the attribute can remain unchanged.

Reply:
In that case can i remove both parameter sets and instead just write Parameter(mandatory=true) ?

Reply:
And if it's allowed to do so, making changes in existing parametersets, will it be considered as a breaking change?

Reply:
Yes, by omitting the `ParameterSetName = ***` you are telling powershell that every parameter sets owns it.
It's not changing existing param set, so is not a breaking change.